frontend/src/data/tasks.json

```diff
@@ -1167,13 +1149,13 @@
             "titleFr": "Pensions publiques"
           }
         ],
-        "titleEn": "Consider impact of your work income as you collect your pensions",
-        "titleFr": "Tenez compte de l'impact qu'a votre revenu de travail lorsque vous touchez vos pensionss"
+        "titleEn": "Consider impact of your work income as you collect your public pensions",
+        "titleFr": "Tenez compte de l'impact qu'a votre revenu de travail lorsque vous touchez vos pensions publiques"
       },
       {
         "answerKey": "unsure-income",
         "descriptionEn": "More Canadians are deciding to work or earn money in their retirement. There are some important things to keep in mind if you earn money while taking your OAS pension:\n\n- Your eligibility for the Guaranteed Income Supplement is based on your income. If your income is higher than a certain amount, you will not qualify for this benefit.\n- If your net world income exceeds the threshold amount ($81,761 for 2022), you have to repay part or your entire OAS pension. Your net world income includes all income you receive in any country, including retirement income from your pensions and RRSPs.",
-        "descriptionFr": "De plus en plus de Canadiens décident de travailler ou de gagner de l'argent pendant leur retraite. Si vous gagnez de l'argent pendant que vous touchez votre pension de la Sécurité de la vieillesse (SV), vous devez tenir compte de certains éléments importants\u00a0:\n\n-  Votre admissibilité au Supplément de revenu garanti dépend de votre revenu. Si vos revenus dépassent un certain montant, vous n'aurez pas droit à cette prestation.\n- Si votre revenu net de toutes provenances est supérieur à un montant seuil (81 761 $ pour 2022), vous devez rembourser une partie ou la totalité de votre pension de la SV. Votre revenu net de toutes provenances comprend tous les revenus que vous recevez dans n'importe quel pays, y compris les revenus de retraite provenant de vos pensions et de vos REER.",
+        "descriptionFr": "De plus en plus de Canadiens décident de travailler ou de gagner de l'argent pendant leur retraite. Si vous gagnez de l'argent pendant que vous touchez votre pension de la Sécurité de la vieillesse (SV), vous devez tenir compte de certains éléments importants\u00a0:\n\n- \u00a0Votre admissibilité au Supplément de revenu garanti dépend de votre revenu. Si vos revenus dépassent un certain montant, vous n'aurez pas droit à cette prestation.\n- Si votre revenu net de toutes provenances est supérieur à un montant seuil (81\u00a0761\u00a0$ pour 2022), vous devez rembourser une partie ou la totalité de votre pension de la SV. Votre revenu net de toutes provenances comprend tous les revenus que vous recevez dans n'importe quel pays, y compris les revenus de retraite provenant de vos pensions et de vos REER.",
```

Comment:
Non-breaking space `\u00a0` is not needed before `Votre admissibilité`.

Reply:
removed `\u00a0 ` in front of `Votre admissibilité`